staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -657,14 +691,26 @@ func (c *Cacher) processEvent(event *watchCacheEvent) {
 	c.incoming <- *event
 }
 
+func channelUsage(cap, len int) float64 {
```

Comment:
Probably shouldn't use cap or len as parameter names since they clash with built-in golang function names.